server/datastore/mysql/migrations/tables/migration_test.go

```diff
@@ -155,6 +155,36 @@ func insertQuery(t *testing.T, db *sqlx.DB) uint {
 	return uint(id) //nolint:gosec // dismiss G115
 }
 
+func checkCollation(t *testing.T, db *sqlx.DB) {
+	type collationData struct {
+		CollationName    string `db:"COLLATION_NAME"`
+		TableName        string `db:"TABLE_NAME"`
+		ColumnName       string `db:"COLUMN_NAME"`
+		CharacterSetName string `db:"CHARACTER_SET_NAME"`
+	}
+
+	stmt := `
+SELECT
+	TABLE_NAME, COLLATION_NAME, COLUMN_NAME, CHARACTER_SET_NAME 
+FROM information_schema.columns
+WHERE
+	TABLE_SCHEMA = (SELECT DATABASE()) 
+	AND (CHARACTER_SET_NAME != ? OR COLLATION_NAME != ?)`
```

Comment:
It's interesting that we don't have to ignore columns with no character set here (aka `NULL`) like anything numeric. 

But I guess it makes sense,
```
> SELECT 'test' != NULL;
+----------------+
| 'test' != NULL |
+----------------+
|           NULL |
+----------------+
```
so rows where the compared value is NULL will not match the condition. That's why we have to use `IS NULL` or `IS NOT NULL` in where conditions when we want that behavior.